dataverse-persistence/src/main/resources/db/migration/V18__addedConsentEntities.sql

```diff
@@ -0,0 +1,35 @@
+create table consent
+(
+    id serial not null
+        constraint consent_pkey
+            primary key,
```

Comment:
I beleive you can omit `not null` when using `primary key`.

Reply:
Yeah, but it is also a pattern in migration sql's. Also having additional security restriction is not a negative Imo.

Reply:
It doesn't add anything since `primary key` force `not null` on a variable, but since it doesn't brake anything we can leave it be.